azerothcore/mod-aoe-loot
# chore. Restructure directory files

## Changes Proposed:
- Create the .h file with all the methods and hooks used
- Modify the information in the README

## Issues Addressed:

- Closes

## SOURCE:


## Tests Performed:

- Windows 10
- In-game


## How to Test the Changes:


1. Download the changes
2. Check that everything is working correctly
